packages/legend-graph/src/models/protocols/pure/v1/engine/generation/V1_FileGenerationInput.ts

```diff
@@ -37,7 +37,7 @@ export class V1_GenerateFileInput {
   static readonly serialization = new SerializationFactory(
     createModelSchema(V1_GenerateFileInput, {
       clientVersion: optional(primitive()),
-      model: V1_pureModelContextDataPropSchema,
+      model: V1_pureModelContextTextPropSchema,
```

Comment:
that would mean here we can use `V1_serializePureModelContext`

Reply:
I think we may want to precisely use PureModelContextText here; otherwise, this would mean that we need a mechanism (a prop schema) to deserialize from any unknown subtype of PureModelContext.